analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/AnalysisVisitor.java

```diff
@@ -239,11 +239,11 @@ public List<Description> visit(JavadocComment n, Analyzer arg) {
     StringBuilder returns = new StringBuilder();
     Map<String, String> commentParams = new LinkedHashMap<>();
     Map<String, String> commentTypeParams = new LinkedHashMap<>();
-    // This regex splits the sentence into 2 if it ends with a . is followed by empty space.
-    String[] sentences = CommentHelperMethods.extractSummary(n).split("\\.\\s+", 2);
-    // Give back the . that is removed by the split.
-    String summary = (sentences.length > 0) ? sentences[0]+"." : null;
-    String remarks = (sentences.length > 1) ? sentences[1] : null;
+    // This regex splits the sentence into 2 at the first dot(.) it finds.
+    String[] sentences = CommentHelperMethods.extractSummary(n).split("\\.", 2);
+    // Add a dot(.) at the end if it is missing.
+    String summary = (sentences.length > 0) ? sentences[0].strip().concat(".") : null;
+    String remarks = (sentences.length > 1) ? sentences[1].strip() : null;
```

Comment:
Can you rebase this PR on `main`, without the changes to comments? That way, we can merge the two things separately.